nodejs/week1/avg.js

```diff
@@ -0,0 +1,14 @@
+//console.log(process.argv);
+
+const arguments = process.argv.slice(2); // we use the slice function to get rid of the first two elements
+const sum = arguments.reduce((a, b) => a + parseInt(b), 0); // reduce to a single number, parse the arg into numbers before trying to do the sum
```

Comment:
Its a good practise to use let instead of const if you know that value will be reassigned later.
let sum = arguments.reduce((a, b) => a + parseInt(b), 0);